CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Mark `initialValues` as immutable in `<LocationForm>`. Fixes STSMACOM-679.
 * Fix `<SearchAndSort>` query doesn't reset when clicking back in browser. Fixes STSMACOM-676.
 * Copy query into search field when navigating back and forth. Fixes STSMACOM-681.
+* CLONE - browse contributors with special characters shows incomplete error message. Refs STSMACOM-684.
```

Comment:
Describe the solution (what changed), not the problem. This message parrots the problem from the ticket, which is totally disconnected from the implementation here, i.e. this change has nothing to do with special characters or error messages. Instead, describe the solution, e.g. "Accept `validateSearchOnSubmit` prop to prevent invalid searches"